### PR TITLE
Fix display notification

### PR DESCRIPTION
**Pull request by khieutrung12**
## WHAT this PR does?

Fix count notifications
Add ringtone

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

File: resources/js/notification_order_status.js [hunk 1]
```diff
@@ -6,17 +6,29 @@ window.Echo = new Echo({
...
 
+var countAudio = 0;
+$(document).mousemove(function(){
+    $(document).click(function(){
+            if(countAudio == 0){
+                document.getElementById('audio').play();
+                countAudio+=1;
+            }
+    })
+})
```
Comment (ghost) [inline resources/js/notification_order_status.js]:
```suggestion
$(document).mousemove(function() {
    $(document).click(function() {
        if (countAudio == 0) {
            document.getElementById('audio').play();
            countAudio+=1;
        }
    })
})
```
indent 4 spaces, fix tương tự các chỗ khác

**Comment by khieutrung12** [comment]
ready

File: resources/js/notification_order_status.js [hunk 2]
```diff
@@ -6,18 +6,30 @@ window.Echo = new Echo({
...
 
+var countAudio = 0;
+$(document).mousemove(function(){
+    $(document).click(function(){
+        if(countAudio == 0){
+            document.getElementById('audio').play();
+            countAudio+=1;
+        }
+    })
+})
```
Comment (ghost) [inline resources/js/notification_order_status.js]:
```suggestion
    $(document).click(function() {
        if (countAudio == 0) {
            document.getElementById('audio').play();
            countAudio += 1;
        }
    })
})
```

File: resources/js/notification_order_status.js [hunk 3]
```diff
@@ -6,18 +6,30 @@ window.Echo = new Echo({
...
-          .notification((notification) => {
-                showNotifications(notification, '#notifications');
-                var notificationsWrapper = $('.dropdown-notifications');
-                var notificationsToggle = notificationsWrapper.find('a[data-toggle]');
-                var notificationsCountElem = notificationsToggle.find('i[data-count]');
-                var notificationsCount = parseInt(notificationsCountElem.data('count'));
-                notificationsCount += 1;
-                notificationsCountElem.attr('data-count', notificationsCount);
-            });
+    if(userID) {
```
Comment (ghost) [inline resources/js/notification_order_status.js]:
```suggestion
    if (userID) {
```

**Comment by khieutrung12** [comment]
ready

**Pull request merged**